gwpy/segments/tests/test_segments.py

```diff
@@ -25,7 +25,9 @@
 
 import h5py
 
-from ...testing.utils import (assert_segmentlist_equal, TemporaryFilename)
+from ...testing.utils import (assert_array_equal, assert_segmentlist_equal,
+                              TemporaryFilename)
+from ...table import Table
```

Comment:
```suggestion
from astropy.table import Table
```
(with appropriate repositioning)